src/aux-server/aux-web/aux-auth/site/AuthApp/AuthApp.vue

```diff
@@ -167,13 +167,13 @@
             </md-dialog-actions>
         </md-dialog>
 
-        <md-dialog :md-active.sync="showCreateRecord">
+        <md-dialog :md-active.sync="showCreateRecord" @keypress.enter="createRecord()">
```

Comment:
@TroyceGowdy This dialog has a form that should catch when people use the enter key to submit the form. Is this change necessary or am I incorrect?